raiden-ts/src/raiden.ts

```diff
@@ -1399,6 +1429,37 @@ export class Raiden {
     return promise;
   }
 
+  /**
+   * Complete a planned UDC withdraw and get the deposit to account.
+   *
+   * Maximum 'value' is the one from current plan, attempting to withdraw a larger value will throw
```

Comment:
The value of a plan can actually not shrink, can't it? So how would we handle a follow up withdraw? Would the transaction just fail for wrong parameters as our code can't catch this early?

Reply:
The plan amount can't shrink, but you request to withdraw anything in `0 < amount <= plan.amount` range, that's what I've tried to describe on the docs and conditions.

Reply:
Okay, Andre explained me that the plan will get deleted by each withdraw. So this is perfect fine.